hack/single-node/bootkube-up

```diff
@@ -9,12 +9,11 @@ fi
 
 # Render assets
 if [ ! -d "cluster" ]; then
-  ../../_output/bin/${local_os}/bootkube render --asset-dir=cluster --api-servers=https://172.17.4.100:443
+  ../../_output/bin/${local_os}/bootkube render --asset-dir=cluster $BOOTKUBE_RENDER_FLAGS
```

Comment:
Is this variable for being able to override settings? These scripts are set up such that you should not have to provide any config, they should "just work" with the vagrant setups. If you want/need the ability to configure we should at least provide a default.